components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -365,4 +365,12 @@ export class TeamDBImpl implements TeamDB {
             repo.save(team);
         }
     }
+
+    public async someOrgWithSSO(): Promise<boolean> {
```

Comment:
Would it be possible to, instead of hacking a string-based query in there, make this a call to the OICD service? Or are we blocked there due to some technical limitation? :thinking:

Reply:
But isn't that a request on Org resources? Here the question is about existence of an Org with a condition. 

Then it would be a PAPI request, right?
This might need to delegate to server to read the current user, right?
Which is the API method calling `someOrgWithSSO`, so it reads like a circle.

Reply:
Yes, should be a regular CreateTeam (CreateOrg) RPC agains public api

Reply:
@easyCZ, same as https://github.com/gitpod-io/gitpod/pull/17303/files#r1173559505

Reply:
That was some confusion about the name of the function. I changed to `someOrgWithSSOExists`.